site/download.md

```diff
@@ -24,29 +24,18 @@ layout: content
 
 -->
 
+{% capture root_url %}http://www.apache.org/dist/incubator/pulsar/pulsar-{{ site.current_version }}/apache-pulsar-{{ site.current_version }}{% endcapture %}
```

Comment:
the URL for current release needs to be `http://www.apache.org/dyn/closer.cgi/...` to point users to the closest mirror.